src/Illuminate/Http/Response.php

```diff
@@ -9,13 +9,32 @@
 use Illuminate\Support\Traits\Macroable;
 use JsonSerializable;
 use Symfony\Component\HttpFoundation\Response as BaseResponse;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
 
 class Response extends BaseResponse
 {
     use ResponseTrait, Macroable {
         Macroable::__call as macroCall;
     }
 
+    /**
+     * Create a new HTTP reponse.
+     *
+     * @param  mixed  $content
+     * @param  int   $status
+     * @param  array  $headers
+     * @return void
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function __construct($content = '', $status = 200, array $headers = [])
```

Comment:
@GrahamCampbell Why did we duplicate the same constructor body as the Symfony Response?

Reply:
It's not a duplication. It removes the types so that content doesn't have to be a string.